sdk/tables/Azure.Data.Tables/src/TableClientOptions.cs

```diff
@@ -16,7 +16,6 @@ public class TableClientOptions : ClientOptions
         /// library.
         /// </summary>
         private const ServiceVersion Latest = ServiceVersion.V2019_02_02;
-        internal static TableClientOptions Default { get; } = new TableClientOptions();
```

Comment:
Is this intentional?

Reply:
Yep, conflicts with the new Core property and is not used.